### PR TITLE
Add a new overload of the `Configuration` method

### PR DESCRIPTION
**Pull request by 0xced**
This new overload is taking an explicit collection of configuration assemblies.

### CODE REVIEW

File: src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs [hunk 1]
```diff
@@ -169,5 +170,39 @@ public static class ConfigurationLoggerConfigurationExtensions
...
             return settingConfiguration.Settings(new ConfigurationReader(configSection, assemblyFinder, configuration: null));
         }
+
+        /// <summary>
+        /// Reads logger settings from the provided configuration object using the provided section name.
+        /// </summary>
+        /// <param name="settingConfiguration">Logger setting configuration.</param>
+        /// <param name="configuration">A configuration object which contains a Serilog section.</param>
+        /// <param name="sectionName">A section name for section which contains a Serilog section.</param>
+        /// <param name="assemblies">A collection of assemblies that contains sinks and other Types.</param>
```
Comment (sungam3r) [inline src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs:180]:
```suggestion
        /// <param name="assemblies">A collection of assemblies that contains sinks and other types.</param>
```

File: src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs [hunk 2]
```diff
@@ -169,5 +170,39 @@ public static class ConfigurationLoggerConfigurationExtensions
...
+
+            return settingConfiguration.Settings(new ConfigurationReader(configuration.GetSection(sectionName), assemblies, new ResolutionContext(configuration)));
+        }
+
+        /// <summary>
+        /// Reads logger settings from the provided configuration object using the default section name.
+        /// </summary>
+        /// <param name="settingConfiguration">Logger setting configuration.</param>
+        /// <param name="configuration">A configuration object which contains a Serilog section.</param>
+        /// <param name="assemblies">A collection of assemblies that contains sinks and other Types.</param>
```
Comment (sungam3r) [inline src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs:200]:
```suggestion
        /// <param name="assemblies">A collection of assemblies that contains sinks and other types.</param>
```

File: src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs (hunk 1 above)
Comment (0xced) [reply src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs]: Applied in 7a79530482e3cadf04902bcf7b9a05ff90f8df9a

File: src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs (hunk 2 above)
Comment (0xced) [reply src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs]: Applied in 7a79530482e3cadf04902bcf7b9a05ff90f8df9a

**Comment by nblumhardt** [comment]
Great, thanks! 👍

**Pull request merged**